roaring/README.md

```diff
@@ -0,0 +1,30 @@
+# The Fuzzer
+
+For complete documentation on go-fuzz, please see: https://github.com/dvyukov/go-fuzz
+
+The fuzzer in relation to the roaring package checks the `Bitmap.UnmarshalBinary` function found in `roaring.go`. In order to use the fuzzer, you can follow these steps:
+
+`cd $GOPATH/src/github.com/pilosa/pilosa/roaring`
+
+`go-fuzz-build ./`
+
+You must now make the workdir/corpus directory. This is achieved by:
+
+`mkdir workdir/corpus`
+
+The fuzzer needs some input to start the fuzzing with. Copy some sample fragments into the workdir/corpus folder. Once you have copied your sample inputs, you are ready to run the fuzzer:
```

Comment:
I don't imagine we will have many (other) newcomers looking at this, but just in case, this could be a bit more clear. I'd say "Pilosa fragments", and maybe add an example line like `cp ~/.pilosa/my-index/my-field/views/standard/fragments/0 workdir/corpus".

Reply:
I made these changes, but they were pushed to pull request #2004.